build.sbt

```diff
@@ -51,7 +51,7 @@ inThisBuild(
     tlBaseVersion          := "0.1",
     tlUntaggedAreSnapshots := false,
     tlFatalWarnings        := sys.env.get("GITHUB_ACTIONS").contains("true"),
-    tlJdkRelease           := Some(17),
+    tlJdkRelease           := Some(8),
```

Comment:
Perhaps could use:
```scala
githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"), JavaSpec.temurin("8"))
```
to ensure potential compatibility errors with Java 8 are caught during the build?

Docs for `githubWorkflowJavaVersions` mention that it will publish for the _first_ in the list: https://typelevel.org/sbt-typelevel/gha.html.

Reply:
Looking into it, we should be ok with `githubWorkflowJavaVersions` as JDK 17. We'll still release for JDK 8 with the `tlJdkRelease` flag.

We'll have a large build matrix with JS and Native (once support is added), so would rather not add extra JDKs unless it's necessary.